Branch-SDK/src/io/branch/referral/BranchRemoteInterface.java

```diff
@@ -270,23 +272,26 @@ public void sendLog(String log) {
 		}
 	}
 	
-	private String convertJSONtoString(JSONObject json) {
+	private String convertJSONtoString(JSONObject json) throws JSONException {
```

Comment:
Myself I really don't like to return null on an exception; surely you could return null instead.